File: sdk/sdk-api-php7/testvo/BatchRespVo.php
<?php
namespace xpay;


class BatchRespVo
{
    private $order_status;
    private $total_count;
    private $single_List;

    /**
     * @return mixed
     */
    public function getOrderStatus()
    {
        return $this->order_status;
    }

    /**
     * @param mixed $order_status
     */
    public function setOrderStatus($order_status): void
    {
        $this->order_status = $order_status;
    }

    /**
     * @return mixed
     */
    public function getTotalCount()
    {
        return $this->total_count;
    }

    /**
     * @param mixed $total_count
     */
    public function setTotalCount($total_count): void
    {
        $this->total_count = $total_count;
    }

    /**
     * @return mixed
     */
    public function getSingleList()
    {
        return $this->single_List;
    }

    /**
     * @param mixed $single_List
     */
    public function setSingleList($single_List): void
    {
        $this->single_List = $single_List;
    }


}

File: sdk/sdk-api-java/src/test/java/com/xpay/sdktest/BatchRespVo.java
package com.xpay.sdktest;

import java.util.List;

public class BatchRespVo {
    private String order_status;
    private String total_count;
    private List<SingleRespVo> single_List;

    public String getOrder_status() {
        return order_status;
    }

    public void setOrder_status(String order_status) {
        this.order_status = order_status;
    }

    public String getTotal_count() {
        return total_count;
    }

    public void setTotal_count(String total_count) {
        this.total_count = total_count;
    }

    public List<SingleRespVo> getSingle_List() {
        return single_List;
    }

    public void setSingle_List(List<SingleRespVo> single_List) {
        this.single_List = single_List;
    }
}
